packages/apps/plugins/plugin-native/src/NativePlugin.tsx

```diff
@@ -94,3 +115,10 @@ export const safeParseJson = <T extends object>(data: string | undefined | null,
   }
   return defaultValue;
 };
+
+export const NativePlugin = (): PluginDefinition => ({
+  meta,
+  ready: async (plugins) => {
+    void initializeNativeApp(plugins);
```

Comment:
I'm wondering if it actually makes sense to await this? Presumably there's functionality in here that would make the app behave weird if it wasn't available?